README.md

```diff
@@ -4,25 +4,78 @@
 
 # DevExpress Reporting for .NET MAUI - Generate Bound-Data Report
 
-...
+This example generates a data-bound report in the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and displays it in the [PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer). To use Reporting APIs in a .NET MAUI project, you need a license that includes [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/).
```

Comment:
In the description, you skip the part that actually creates the report layout. The first step already creates an `XtraReportInstance` object... But that object is a report with predefined layout. And to understand what this report does, our customers would need to review files in the **ReportLibrary** folder. 

We can go two different ways: 
- Mention the ReportLibrary folder and say that they can review those files and that this is where `XtraReportInstance` is defined. 
- Describe the entire process of how to create a report. 

I'm guessing you don't want to describe how to create a report, so we'll go with the first option. 

```suggestion
This example uses [DevExpress PDF Viewer for .NET MAUI](https://docs.devexpress.com/MAUI/404632/pdf-viewer/pdf-viewer) to display a data-bound report in a mobile app. 

You can adjust this example to use any previously-defined report. In this repository, the **ReportLibrary** folder contains the data source and the report layout definition. We constructed the report with the help of the [DevExpress Report Designer](https://docs.devexpress.com/MAUI/404437/reporting/reporting-overview) and named the report class `XtraReportInstance`. The code below creates this object to generate a report document. 

**NOTE**: To use Reporting APIs in a .NET MAUI project, you need a license that includes [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/).
```